pom.xml

```diff
@@ -181,7 +181,7 @@
         <dependency>
             <groupId>org.igniterealtime.openfire</groupId>
             <artifactId>xmppserver</artifactId>
-            <version>4.7.4</version>
+            <version>4.8.1</version>
```

Comment:
I _think_ this also works with 4.8.0 instead of 4.8.1? It might be good to be as liberal as possible.